carwash/advancedCarWash/carGenerator.cc

```diff
@@ -0,0 +1,60 @@
+/**
+Car Generator for carWash simulator
+
+This component generates "cars" which are sent to the carwash
+*/
+
+#include <sst/core/sst_config.h>
+#include <sst/core/interfaces/stringEvent.h>
+#include "carGenerator.h"
```

Comment:
It's encouraged to include the associated header file for a cc file right after sst_config.h.  There are other instances of this throughout the codebase.